### PR TITLE
fix broken timers on the new tagged measurements

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

**Comment by netroy** [comment]
wrote them as 3 commit to TDD the behavior.. the tests fail on the second commit. Please feel free to squash them when merging.

**Comment by bryanmikaelian** [comment]
@netroy Thanks for the PR. Just to confirm, are you only seeing broken timers for the `sum` aggregate?

**Comment by bryanmikaelian** [comment]
Nailed down the commit where this broke: https://github.com/librato/statsd-librato-backend/commit/8bc40dc3b01019489b08d07f2756c51ec827cbad#diff-2ae335a2155c51431a9d83207bf6f5fbL197

**Comment by netroy** [comment]
@bryanmikaelian yep..  since by default we see the average of sums, unless I group things, I only see flat lines.

File: lib/librato.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ var timer_gauge_pct = function(timer_name, values, pct, suffix)
   var max = values[numInThreshold - 1];
   var min = values[0];
 
-  var sum = 0;
+  var sum = values.reduce(function (s, current) { return s + current }, 0);
```
Comment (netroy) [inline lib/librato.js]: should this only sum the first `numInThreshold` values ?

Comment (bryanmikaelian) [inline lib/librato.js]: Yea that should be the expected behavior.

**Comment by netroy** [comment]
@bryanmikaelian updated

**Comment by bryanmikaelian** [comment]
🆒  Will get this merged and released to npm. Thanks @netroy. Great catch! 🐟

**Comment by netroy** [comment]
most welcome, thanks for the quick review .. cheers 😄 
would it be possible to have an npm release today? if not, i can use the git url temporarily.

**Comment by bryanmikaelian** [comment]
Absolutely! Publishing now. Should be available soon.

**Pull request merged**